### PR TITLE
[full-ci][tests-only][e2e] Backport e2e test for share decline

### PR DESCRIPTION
**Pull request by amrita-shrestha**
## Description
This PR backport https://github.com/owncloud/web/pull/8142 pr changes from master to stable

## Related Issue

- Part of https://github.com/owncloud/web/pull/8142

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: tests/e2e/support/objects/app-files/link/actions.ts [hunk 1]
```diff
@@ -178,7 +185,7 @@ export const addExpiration = async (args: addExpirationArgs): Promise<void> => {
 
   await page.locator(linkExpiryDatepicker).evaluate(
     (datePicker: any, { newExpiryDate }): any => {
-      datePicker.__vue__.updateValue(newExpiryDate)
+      datePicker?.__vueParentComponent?.refs?.calendar.move(newExpiryDate)
```
Comment (saw-jan) [inline tests/e2e/support/objects/app-files/link/actions.ts:188]: I think the PR (https://github.com/owncloud/web/pull/81280 that changes this is not ported to stable and I guess the PR is only for the master (for next version)

Comment (amrita-shrestha) [reply tests/e2e/support/objects/app-files/link/actions.ts:188]: I think its ported because ci passed

Comment (saw-jan) [reply tests/e2e/support/objects/app-files/link/actions.ts:188]: No, it's not.
Maybe this works with existing stable code. But anyways, it's good.

**Review by saw-jan — COMMENTED** [review]
LGTM

**Review by saw-jan — APPROVED** [review]

**Review by SagarGi — COMMENTED** [review]
LGTM :+1:

**Review by individual-it — APPROVED** [review]

File: tests/e2e/support/objects/app-files/link/actions.ts (hunk 1 above)
Comment (individual-it) [reply tests/e2e/support/objects/app-files/link/actions.ts:188]: please make sure that the code is as much as possible same between master and stable branch

**Pull request merged**